apollo-router/Cargo.toml

```diff
@@ -127,6 +127,7 @@ tracing-subscriber = { version = "0.3.11", features = ["env-filter", "json"] }
 url = { version = "2.2.2", features = ["serde"] }
 urlencoding = "2.1.0"
 yaml-rust = "0.4.5"
+access-json = "0.1.0"
```

Comment:
NIT: I'd feel more comfortable if this was at a higher version level.

Reply:
Yes I know, BTW I plan to contribute to this project to add some features (like the `*` syntax)